ts/packages/node/src/stores/CsvFile.ts

```diff
@@ -0,0 +1,23 @@
+import { type Config } from '@configu/ts';
+import { parse, stringify } from 'csv/sync';
+import { FileConfigStore } from './File';
+
+export type CsvFileConfigStoreConfiguration = { path: string };
+
+export class CsvFileConfigStore extends FileConfigStore {
+  constructor({ path }: CsvFileConfigStoreConfiguration) {
+    const initialFileState = '';
+    super('csv-file', { path, initialFileState });
+  }
+
+  parse(fileContent: string): Config[] {
+    const csvRows = parse(fileContent);
+    const configs = csvRows.slice(1).map(([set, key, value]: [string, string, string]) => ({ set, key, value }));
+    return configs;
+  }
+
+  stringify(nextConfigs: Config[]): string {
+    const csvRows = [['set', 'key', 'value'], ...nextConfigs.map(({ set, key, value }) => [set, key, value])];
+    return stringify(csvRows);
+  }
```

Comment:
We should utilize the capabilities of the `csv` package to streamline these methods:

```suggestion
  parse(fileContent: string): Config[] {
    return parse(fileContent, { columns: true });
  }

  stringify(nextConfigs: Config[]): string {
    return stringify(nextConfigs, { header: true });
  }
```